src/clients/PublicClient.ts

```diff
@@ -340,6 +340,29 @@ class PublicClient extends BaseClient {
     });
     return hexToNumber(res);
   }
+
+  /**
+   * Returns all tokens by the given address.
+   * @param address The address of the account or contract.
```

Comment:
Why are accounts and contracts distinguished here? In =nil;, any wallet is a contract, and vice versa, no?